tests/test_mypy.yml

```diff
@@ -0,0 +1,26 @@
+# Tests for marshmallow_dataclass.mypy, using pytest-mypy-plugins
+# NOTE: Since pytest-mypy-plugins is not installed on pypy (see setup.py), these tests
+#       will not even be collected by pytest
+- case: NewType_basic
+  mypy_config: |
+    follow_imports = silent
+    plugins = marshmallow_dataclass.mypy
+  main: |
+    from dataclasses import dataclass
+    import marshmallow as ma
+    from marshmallow_dataclass import NewType
+
+    Email = NewType("Email", str, validate=ma.validate.Email)
+    UserID = NewType("UserID", validate=ma.validate.Length(equal=32), typ=str)
+
+    @dataclass
+    class User:
+        id: UserID
+        email: Email
+
+    user = User(id="a"*32, email="user@email.com")
+    reveal_type(user.id)  # N: Revealed type is 'builtins.str'
```

Comment:
Neat!

Reply:
Indeed! It does make some lines a bit long, but still better than referencing invisible line numbers IMO.